plugins/modules/inventory_intent.py

```diff
@@ -2598,6 +2622,14 @@ def get_diff_merged(self, config):
                                         self.msg = "Device Updation get failed"
                                     self.log(self.msg, "ERROR")
                                     break
+                                elif execution_details.get("endTime"):
+                                    self.status = "success"
+                                    self.result['changed'] = True
+                                    self.result['response'] = execution_details
+                                    self.msg = "Devices {0} present in Cisco Catalyst Center and updated successfully".format(str(device_to_update))
```

Comment:
"Devices {0} are present in the Cisco Catalyst Center and have been updated successfully".format(str(device_to_update))

Reply:
added the log message with device ip.